src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/AzureRM.DataLakeAnalytics.psd1

```diff
@@ -0,0 +1,90 @@
+#  
+# Module manifest for module 'Microsoft.Azure.Commands.DataLakeAnalytics'  
```

Comment:
All RM .psd1 files start with AzureRM.
see: AzureRM.RedisCache.psd1, AzureRM.Profile.psd1, etc. I am not sure what other projects you are referring to that do not follow the "AzureRM.<feature>.psd1" format.